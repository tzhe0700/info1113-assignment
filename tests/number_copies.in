ADD SPELLBOOK spellbooks.csv 111111
ADD SPELLBOOK spellbooks.csv 111112
ADD SPELLBOOK spellbooks.csv 111113
NUMBER COPIES
EXIT

