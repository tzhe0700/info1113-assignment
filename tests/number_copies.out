user: Successfully added: Advanced Potion Making (Libatius Borage).

user: No such spellbook in file.

user: No such spellbook in file.

user: Advanced Potion Making (Libatius Borage): 1

user: Ending Archive process.

